runtime/compiler/optimizer/J9Optimizer.cpp

```diff
@@ -403,6 +409,7 @@ static const OptimizationStrategy reducedWarmStrategyOpts[] =
    { OMR::deadTreesElimination,                      OMR::IfEnabled                  }, // cleanup at the end
    { OMR::jProfilingRecompLoopTest,                  OMR::IfLoops                    },
    { OMR::cheapTacticalGlobalRegisterAllocatorGroup, OMR::IfEnabled                  },
+   { OMR::treeLowering,                              OMR::MustBeDone                 },
```

Comment:
Is `reducedWarmStrategyOpts` ever used? Or how is it used? I just noticed it doesn't have `globalLiveVariablesForGC` before endOpts. 
Would there be an issue without `globalLiveVariablesForGC`? Same question applies to the change for `hotStrategyOpts`, `scorchingStrategyOpts`, and `AOTStrategyOpts`.

Reply:
> Is reducedWarmStrategyOpts ever used?

I'm not sure. I actually had the same question but haven't had the chance to investigate yet.

> Would there be an issue without globalLiveVariablesForGC? Same question applies to the change for hotStrategyOpts, scorchingStrategyOpts, and AOTStrategyOpts.

We definitely use these strategies, so I don't think it's a problem.

Reply:
I don't think `reducedWarmStrategyOpts` is used, and it doesn't appear to have been used for many years.  I think it can be removed, but either in a separate PR or in a different commit.

FYI @vijaysun-omr or @mpirvu in case you have reason to pardon it.

`globalLiveVariablesForGC` is done as part of the `finalGlobalGroup` that is included in the hot, scorching, and AOT strategies.

Reply:
```
// ***************************************************************************
// A (possibly temporary) strategy for partially optimizing W-Code
// ***************************************************************************
//
static const OptimizationStrategy reducedWarmStrategyOpts[] =
```
Indeed, this was only used by the static compiler, so it can be removed as far I am concerned.

Reply:
I've opened #12029 to track removing `AOTStrategyOpts` and `reducedWarmStrategyOpts` as both have been used for years.